### DISCUSSION TITLE
Do you have a discord server?

### DISCUSSION THREAD

**Discussion by ghost** [CosmicHikari/Weather2-Remastered#16] [Questions] [answered]
Hi!

I wanted to ask you, do you have a discord server regarding the weather2 remaster? It would allow us to exchange ideas and follow the progress of your project.

**Comment by Mrbt0907** [comment, marked as answer]
I didn't make a server for weather2 just yet. Depending on how this project goes, I may make a server in the future.

**Reply by CosmicHikari** [reply]
Got my vote for making one.
